### PR TITLE
Adding tao verification

### PR DESCRIPTION
**Pull request by Aryess**
PR for https://github.com/CoderDojo/community-platform/issues/741

### CODE REVIEW

File: web/public/templates/dojos/dojo-detail.dust [hunk 1]
```diff
@@ -8,6 +8,7 @@
             <b ng-bind-html="dojo.name"></b>
             <span ng-if="dojo.verified === 1" class="verification-icon"><span class="cd-success-text glyphicon glyphicon-ok-sign"></span> {@i18n key=" Verified!"/}</span>
             <span ng-if="dojo.verified != 1" class="verification-icon"><span class="cd-warning-text glyphicon glyphicon-remove-sign"></span> {@i18n key=" Not verified"/}</span>
+            <span ng-if="dojo.taoVerified === 1" class="verification-icon"><span class="cd-success-text glyphicon glyphicon-ok-sign"></span> {@i18n key=" Tao verified"/}</span>
```
Comment (kennanseno) [inline web/public/templates/dojos/dojo-detail.dust]: won't this create a duplicate icon if both dojo.verified and dojo.taoVerified are === 1 ?

Comment (Aryess) [inline web/public/templates/dojos/dojo-detail.dust]: The icon is the same indeed, but wording is different. As agreed with @tangentfairy, this is more of a placeholder for now, but I'll try to find a better icon

Comment (tangentfairy) [inline web/public/templates/dojos/dojo-detail.dust]: Yeah the placeholder is fine for now, @rosalanghammer is working on a
permanent version!

On Tuesday, January 26, 2016, Yoann Celton <notifications@github.com> wrote:

> In web/public/templates/dojos/dojo-detail.dust
> <https://github.com/CoderDojo/cp-zen-platform/pull/811#discussion_r50795008>
> :
>
> > @@ -8,6 +8,7 @@
> >              <b ng-bind-html="dojo.name"></b>
> >              <span ng-if="dojo.verified === 1" class="verification-icon"><span class="cd-success-text glyphicon glyphicon-ok-sign"></span> {@i18n key=" Verified!"/}</span>
> >              <span ng-if="dojo.verified != 1" class="verification-icon"><span class="cd-warning-text glyphicon glyphicon-remove-sign"></span> {@i18n key=" Not verified"/}</span>
> > +            <span ng-if="dojo.taoVerified === 1" class="verification-icon"><span class="cd-success-text glyphicon glyphicon-ok-sign"></span> {@i18n key=" Tao verified"/}</span>
>
> The icon is the same indeed, but wording is different. As agreed with
> @tangentfairy <https://github.com/tangentfairy>, this is more of a
> placeholder for now, but I'll try to find a better icon
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CoderDojo/cp-zen-platform/pull/811/files#r50795008>.
>


-- 
Ursula Clarke Everett
Technical Lead @ CoderDojo Foundation
S: tangentfairy | T: @CoderDojo | @tangentfairy

-- 
*Volunteer at your local Dojo and inspire the next generation of digital 
creators!*

   - Pledge to start a Dojo <https://coderdojo.com/pledge-a-dojo/>
   - How to volunteer at your local CoderDojo 
   <https://coderdojo.com/get-involved/volunteer/>
   - Read the CoderDojo Foundation Strategy 
   <https://coderdojo.com/news/2015/08/07/the-coderdojo-foundation-strategy-20152016/>


-- 
Hello World Foundation operating as the CoderDojo Foundation is a company 
limited by guarantee not having a share capital with charitable status, CHY 
20812. Registered in Dublin, Ireland, No 524255. Registered Office: 
Dogpatch Labs, Unit 1, CHQ Building, Custom House Quay, Dublin 1, Ireland.

**Comment by tangentfairy** [comment]
Checked out this branch and just added the icon itself in https://github.com/CoderDojo/cp-zen-platform/pull/817 - merged now. Thanks @Aryess !

**Pull request closed without merging**